flixel/util/FlxDirection.hx

```diff
@@ -0,0 +1,50 @@
+package flixel.util;
+
+@:enum abstract FlxDirection(Int) from Int to Int
```

Comment:
I know that the naming convention for enums is singular (for good reason), but this isn't _really_ an enum, it's a bit field, so one value can contain multiple directions. `FlxDirections` makes more sense to me because of that. That also matches the naming convention for bit fields in C#, and maybe other languages.

There should be a doc comment here mentioning that it's a bit field (for people new to the concept so they can look it up), plus `@since 4.10.0`.

The `from Int to Int` is unfortunate for type safety, but necessary backwards compatbility.